src/analysis/souffle/taint.dl

```diff
@@ -77,6 +78,16 @@
 // Rules
 //-----------------------------------------------------------------------------
 
+// Draw an edge from any operation's operand to its result.
+resolvedEdge(owner, src, tgt) :- operationToOperands(owner, _, tgt, src).
+
+// For all operations other than `=`, draw an edge from the special
+// "ArbitraryComputation" `AccessPath` to the result. This prevents default
+// propagation of `IntegrityTag`s through the operation.
+isAccessPath("ArbitraryComputation").
+resolvedEdge(owner, "ArbitraryComputation", tgt) :-
```

Comment:
I have a similar question as above.

Reply:
Discussed this in response in other thread, I think these two comment-points may look strange in isolation but they make sense when considered together.